keda/templates/manager/role.yaml

```
@@ -17,11 +17,13 @@ rules:
   resources:
   - secrets
   verbs:
+  {{- if not (and .Values.permissions.operator.restrict.secret .Values.certificates.certManager.enabled) }}
```

I don't get this, we will add "write" permissions if restrict and certManager are false. So if we se certManager false and restrict true, we won't add writing permissions and the cert-controller will fail generating the secrets.
For adding this group of actions, the restrict flag doesn't matter, as it's for cert-controller and them aren't covered by ClusterRole

```suggestion
  {{- if and .Values.certificates. autoGenerated (not .Values.certificates.certManager.enabled) }}
```